### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by saruuja**
#This is a desktop version project consisting of:-
 three main sections Banner section, Work section, and footer or contact form section

 ##I used to build with this millstone
1.HTML &CSS
2. GitHub flow & Linter

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+            <li class="arrow_one"></li>
+            <li class="Frame">Frameworks</li>
+            <li class="line_one"></li>
+            <li class="arrow_two"></li>
+            <li class="skll">Skills</li>
+            <li class="line_two"></li>
+          </ul>
+        </div>
+      </div>
+      <div class="contact">
```
Comment (topeogunleye) [inline index.html:222]: - [ ] I kindly suggest that you ensure that all the elements of the contact form are centered including the `Get in touch` button. Please kindly check your [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848): 

| Your Design  | Template  Design |
| ------------- | ------------- |
| Screenshot from 2022-06-04 04-48-31 | Screenshot from 2022-06-04 04-49-51  |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @saruuja,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * PR has a good summary description :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
>- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
Screenshot from 2022-04-20 08-55-14PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
>- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
Screenshot from 2022-04-20 08-55-14PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
>- [ ] From now on, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Wrong
Screenshot from 2022-06-04 04-38-26
> ### Good Commit Messages:
>- [x]  Add fix for dark mode toggle state
>- [x] Update README file.
>- [x] Add styles to h1.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (saruuja) [reply index.html:222]: I made a Get it Touch button in the center

**Review by Kandy-Peter — APPROVED** [review]
Hi @saruuja  ,

# APPROVED 🟢 

ObedientGreenAndeancat-max-1mb

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

#### CONGRATULATIONS! 🎉 

##  Optional suggestion

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**